website/src/pages/changelog.mdx

```diff
@@ -18,6 +18,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added support for enabling/disabling swipe gestures for Windows WebView2. Added by @leaanthony in [PR](https://github.com/wailsapp/wails/pull/2878)
 - When building with `-devtools` flag, CMD/CTRL+SHIFT+F12 can be used to open the devtools. Added by @leaanthony in [PR](https://github.com/wailsapp/wails/pull/2915)
 
+### Changed
+
+- AssetServer requests are now processed asynchronously without blocking the main thread on Windows. Changed by @stffabi in [PR](https://github.com/wailsapp/wails/pull/2926)
+- AssetServer requests are now processed concurrently by spawning a goroutine per request. Changed by @stffabi in [PR](https://github.com/wailsapp/wails/pull/2926)
```

Comment:
Duplicate record?
When fixing this please merge it with the other `Changed` section, got mixed with the devtools PR, don't know how it didn't get marked as conflict.

Reply:
That's not a duplicate record, they both have the same prefix so they might look like duplicate but they aren't.

- The first describes that we now don't block the main-thread during the processing of a request on Windows. 
- The second describes we now spawn a goroutine per request, allowing them to be processed concurrently. Previously we had 10 worker goroutines processing requests.

Reply:
PR #2940 is up to fix the duplicate `changed` section that came in with the devtools PR.

Reply:
Merged. Git black magic bites again